src/parsers/xyz.js

```diff
@@ -140,10 +144,37 @@ function fromMaterial(materialOrConfig, fractional = false) {
     return fromBasis(basis, "%11.6f");
 }
 
+/**
+ * Function splits the xyzFile string at new lines and then returns the first element of the array.
+ * The first line of the XYZ file should contain the number of atoms in the structure.
+ * @param {String} xyzFile
+ * @returns {Number}
+ */
+export function xyzFileAtomsCount(xyzFile) {
+    return Integer.parseInt(xyzFile.split(/\r?\n/)[0]);
```

Comment:
Why not just use a standard `parseInt`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt??